wishlistApp/views.py

```diff
@@ -90,15 +90,21 @@ def new_item(request):
             #(not sure if saving the form OR putting info into an Item obj is correct)
             '''
             name = request.POST['itemN']
-            url = request.POST['itemURL']
+            itemURL = request.POST['itemURL']
             desc = request.POST['itemD']
             image = request.POST['imageURL']
             priority = request.POST['itemP']
-            ins = Item(name=name, image=url, description=desc, priority = priority, user_id=request.user.get_username())
+            for urls in URL.objects.all():
+                if urls == itemURL:
+                    ins = Item(name=name, image=url, description=desc, priority = priority, user_id=request.user.get_username(), url_id=urls)
+                    break
+                else:
+                    ins = Item(name=name, image=url, description=desc, priority = priority, user_id=request.user.get_username(), url_id=itemURL)
+                    break
             ins.save()
             '''
             messages.success(request, f"Item has been added to your Wishlist!")
-            context = {
-                'form' : form,
-            }
-            return render(request, 'wishlistApp/newItem.html', context)
+            return render(request, 'wishlistApp/newItem.html', {'form': form})
+        else:
+            form = ItemUpdateForm(request.POST, instance=request.user)
+    return render(request, 'wishlistApp/newItem.html', context)
```

Comment:
not sure what `context` is?? it's not defined in your route. might give you an error once you're able to get your html pages to load

Reply:
fixed it! i completely passed it over when I was working on the first if statement.